libs/core/langchain_core/prompts/chat.py

```diff
@@ -544,7 +544,7 @@ def from_template_file(
         Returns:
             A new instance of this class.
         """
-        template = Path(template_file).read_text()
+        template = Path(template_file).read_text(encoding="utf-8")
```

Comment:
This is a breaking change we can roll this out as part of 0.4 cc @ccurme / @sydney-runkle 

Ideally change default encoding to utf-8 in every instance